app/src/main/java/org/oppia/app/topic/review/TopicReviewFragmentPresenter.kt

```diff
@@ -28,14 +29,16 @@ class TopicReviewFragmentPresenter @Inject constructor(
   private val logger: Logger,
   private val topicController: TopicController
 ) : ReviewSkillSelector {
-
+  private lateinit var topicId: String
   private val routeToReviewListener = activity as RouteToConceptCardListener
 
   private lateinit var reviewSkillSelectionAdapter: ReviewSkillSelectionAdapter
 
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
+    topicId = checkNotNull(fragment.arguments?.getString(TOPIC_ID_ARGUMENT_KEY)) {
+      "Expected topic ID to be included in arguments for TopicTrainFragment."
```

Comment:
Nit: `Expected topic ID to be included in arguments for TopicReviewFragment.`

Reply:
Done